tests/v2/test_storage.py

```diff
@@ -1,96 +1,87 @@
-# import array
+import array
```

Comment:
The story here is that when we first started the v3 work, I tried to "reuse" the v2 store tests. It wasn't a great fit and I wasn't able to migrate many tests to the new API those changes rather than continuing to use this module.